src/charms/carbon/email.cr

```diff
@@ -0,0 +1,8 @@
+# TODO: Move this in to Carbon directly
+# This patch allows us to see what the class names of
+# all the carbon emails are
+abstract class Carbon::Email
+  def self.email_classes
+    {{ Carbon::Email.all_subclasses.select(&.abstract?.!) }}
```

Comment:
First off, I absolutely love this `.!` negation 😂 Thanks @jan-zajic 😁 

Second, I kinda wish this was just already built-in for runtime, but is there a better name I could use? Maybe I just call this method `subclasses`? 🤔